dev-python/pycpio/pycpio-1.1.2.ebuild

```diff
@@ -0,0 +1,18 @@
+# Copyright 2023-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{11..12} )
```

Comment:
Can we add 3.13 to compat?

Reply:
I need to test that first, I'll try to do that today.